cli/ballerina-cli/src/main/java/io/ballerina/cli/task/ListTestGroupsTask.java

```diff
@@ -51,7 +51,7 @@ public ListTestGroupsTask(PrintStream out, boolean displayWarning) {
     public void execute(Project project) {
         for (ModuleId moduleId : project.currentPackage().moduleIds()) {
             Module module = project.currentPackage().module(moduleId);
-            TestProcessor testProcessor = new TestProcessor();
+            TestProcessor testProcessor = new TestProcessor(null);
```

Comment:
Let's keep two constructors. One no-args constructor and another to pass the JarResolver

Reply:
Implemented 2 constructors and fixed `TestProcessor(null)`